xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxAccountServiceRaw.java

```diff
@@ -96,30 +97,115 @@ public FtxAccountServiceRaw(Exchange exchange) {
     }
   }
 
+  public FtxResponse<List<FtxLendingInfoDto>> getFtxLendingInfos(String subaccount)
```

Comment:
should these lending/borrowing services live inside of here as opposed to the lending and borrowing service classes?

Reply:
this was not addressed? these are lending/borrowing apis not account apis

Reply:
I delete borrowing/lending request -> for methods in LendingService and BorrowingService